nexus/inventory/src/builder.rs

```diff
@@ -387,6 +402,105 @@ impl CollectionBuilder {
     pub fn found_error(&mut self, error: InventoryError) {
         self.errors.push(error);
     }
+
+    /// Record information about a sled that's part of the control plane
+    pub fn found_sled_inventory(
+        &mut self,
+        source: &str,
+        inventory: sled_agent_client::types::Inventory,
+    ) -> Result<(), anyhow::Error> {
+        let sled_id = inventory.sled_id;
+
+        // Normalize the baseboard id, if any.
+        use sled_agent_client::types::Baseboard;
+        let baseboard_id = match inventory.baseboard {
+            Baseboard::Pc { .. } => None,
```

Comment:
Why not utilize `identifier` and `model` fields from the `Pc` variant in the same way and then not have the bifurcated code base?

Reply:
(I think this is covered by the other thread on the database schema)